coord/src/module.c

```diff
@@ -1931,7 +1931,15 @@ int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
       tableSize = 16384;
   }
 
-  MRCluster *cl = MR_NewCluster(initialTopology, sf, 2);
+  size_t num_connections_per_shard;
+  if (clusterConfig.connPerShard) {
+    num_connections_per_shard = clusterConfig.connPerShard;
+  } else {
+    // default
+    num_connections_per_shard = RSGlobalConfig.numWorkerThreads + 1;
+  }
```

Comment:
can't that be the default value of `clusterConfig.connPerShard`?

Reply:
not directly, `RSGlobalConfig.numWorkerThreads` default is 0, and only at this point we can check if and what are the values of the configurations.